src/geocel/CMakeLists.txt

```diff
@@ -75,10 +75,14 @@ if(CELERITAS_USE_VecGeom)
   list(APPEND SOURCES
     vg/VecgeomParams.cc
     vg/VecgeomParamsOutput.cc
-    vg/detail/VecgeomNavCollection.cc
     vg/RaytraceImager.cc
     vg/SafetyImager.cc
   )
+  if(VecGeom_NAV STREQUAL "path")
```

Comment:
I don't see a point in preserving the ability to run with VecGeom_NAV in "path" mode, as the performance is probably better in the NavIndex mode.